usr/share/rear/backup/BORG/default/500_make_backup.sh

```diff
@@ -72,5 +102,7 @@ elif is_true "$VERBOSE"; then
 else
     borg_create 0<&6
 fi
+borg_exit_code=$?
 
+check_BORG_create_return_code $borg_exit_code
 StopIfError "Borg failed to create backup archive, borg rc $?!"
```

Comment:
While you are at that code I would appreciate it
if you could by the way replace the 'StopIfError'
function call as a separated command with using
bash directly to call the 'Error' function as
```
check_BORG_create_return_code $borg_exit_code || Error "Borg failed to create backup archive, borg rc $borg_exit_code"
```

For the reason behind see my comments about
"Using the ...IfError functions can result unexpected behaviour in certain cases"
in usr/share/rear/lib/_input-output-functions.sh
that is currently online at
https://github.com/rear/rear/blob/master/usr/share/rear/lib/_input-output-functions.sh#L794

We keep the ...IfError functions in ReaR only because
they are used at many places in our code (more than 200)
and I did not yet find the time to clean that up
(but at some future time I will clean that up).